sdk/iot/Azure.Iot.Hub.Service/src/ExportImportDeviceExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Iot.Hub.Service.Models;
+
+namespace Azure.Iot.Hub.Service
+{
+    /// <summary>
+    /// Helper functions for mutating the <see cref="ExportImportDevice"/> instance.
+    /// </summary>
+    internal static class ExportImportDeviceExtensions
```

Comment:
Was this class handcrafted, or generated by autorest?

Reply:
Handcrafted.